### PR TITLE
chore: remove leftovers from relay upgrade

### PR DESCRIPTION
**Pull request by araujobarret**
The type of this PR is: **Chore**

### Description

Removes Relay ignore leftovers

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by dzucconi — APPROVED** [review]

File: src/Apps/Partner/Components/PartnerArtists/PartnerArtistList/PartnerArtists.tsx [hunk 1]
```diff
@@ -41,7 +41,6 @@ export const PartnerArtists: React.FC<PartnerArtistsProps> = ({
       <PartnerArtistListFragmentContainer
         partnerSlug={slug}
         scrollTo={scrollTo!}
-        // FIXME: RELAY UPGRADE
         artists={artists as any}
```
Comment (dzucconi) [inline src/Apps/Partner/Components/PartnerArtists/PartnerArtistList/PartnerArtists.tsx:44]: Should remove the `any`

File: src/Apps/Order/orderRoutes.tsx [hunk 2]
```diff
@@ -127,7 +127,6 @@ export const orderRoutes: AppRouteConfig[] = [
             { order }
           )
 
-          // FIXME RELAY_UPGRADE
```
Comment (dzucconi) [inline src/Apps/Order/orderRoutes.tsx:130]: I guess this still looks relevant: there's some code here that would need to be walked back.

Comment (damassi) [reply src/Apps/Order/orderRoutes.tsx:130]: Wasn't able to solve this one, but this maintains that the logic we were testing as valid within the test, albeit annoyingly

File: src/Apps/Partner/Components/PartnerArtists/PartnerArtistList/PartnerArtists.tsx (hunk 1 above)
Comment (dzucconi) [reply src/Apps/Partner/Components/PartnerArtists/PartnerArtistList/PartnerArtists.tsx:44]: It should be fine if you use the extractNodes util on the parent connection

**Pull request closed without merging**